src/app/zap-templates/zcl/data-model/chip/time-synchronization-cluster.xml

```diff
@@ -68,42 +101,81 @@ limitations under the License.
     <attribute side="server" code="0x0002" define="TIME_SOURCE" type="TimeSourceEnum" default="0x00" optional="true">TimeSource</attribute>
-    <attribute side="server" code="0x0003" define="TRUSTED_TIME_NODE_ID" type="node_id" writable="true" isNullable="true" optional="false">
-	<description>TrustedTimeNodeId</description>
-	<access op="read" privilege="view"/>
-	<access op="write" privilege="administer"/>
-    </attribute>		
-    <attribute side="server" code="0x0004" define="DEFAULT_NTP" type="CHAR_STRING" length="128" writable="true" isNullable="true" optional="true">
-	<description>DefaultNtp</description>
-	<access op="read" privilege="view"/>
-	<access op="write" privilege="administer"/>
-    </attribute>		
-    <attribute side="server" code="0x0005" define="TIME_ZONE" type="ARRAY" entryType="TimeZoneStruct" min="1" max="2" writable="true" optional="true">
-	<description>TimeZone</description>
-	<access op="read" privilege="view"/>
-	<access op="write" privilege="manage"/>
-    </attribute>
-    <attribute side="server" code="0x0006" define="DST_OFFSET" type="ARRAY" entryType="DSTOffsetStruct" max="20" writable="true" optional="true">
-	<description>DSTOffset</description>
-	<access op="read" privilege="view"/>
-	<access op="write" privilege="manage"/>
-    </attribute>		
-    <attribute side="server" code="0x0007" define="LOCAL_TIME" type="epoch_us" default="0x00" isNullable="true" optional="true">LocalTime</attribute>
-    <attribute side="server" code="0x0008" define="TIME_ZONE_DATABASE" type="boolean" default="0" optional="true">TimeZoneDatabase</attribute>
-    <attribute side="server" code="0x0009" define="NTP_SERVER_PORT" type="INT16U" isNullable="true"  optional="true">NtpServerPort</attribute>
-
-    <!-- Test Commands -->
-    <command source="client" code="0x00" name="SetUtcTime" optional="false">
-	<description>Upon receipt of this command, the server MAY update its UTCTime Attribute to match the time specified in the command</description>
-	<arg name="UtcTime" type="epoch_us"/>	
-	<arg name="Granularity" type="GranularityEnum"/>	
-	<arg name="TimeSource" type="TimeSourceEnum" optional="true"/>
-    </command>	
+    <attribute side="server" code="0x0003" define="TRUSTED_TIME_SOURCE" type="TrustedTimeSourceStruct" isNullable="true" optional="true">TrustedTimeSource</attribute>
+    <attribute side="server" code="0x0004" define="DEFAULT_NTP" type="CHAR_STRING" length="128" isNullable="true" optional="true">DefaultNTP</attribute>
+    <attribute side="server" code="0x0005" define="TIME_ZONE" type="ARRAY" entryType="TimeZoneStruct" optional="true">TimeZone</attribute>
+    <attribute side="server" code="0x0006" define="DST_OFFSET" type="ARRAY" entryType="DSTOffsetStruct" optional="true">DSTOffset</attribute>
+    <attribute side="server" code="0x0007" define="LOCAL_TIME" type="epoch_us" default="0xFFFFFFFFFFFFFFFF" isNullable="true" optional="true">LocalTime</attribute>
```

Comment:
Just as a note: this is really misleading.  This "default" value will be ignored in practice because we are using AttributeAccessInterface, and it does not correctly express what the spec says anyway.... because this is not representing any sort of spec concept to start with.

I don't feel too strongly about this, but it'll really confuse people looking at this in the ZAP UI.

Reply:
> It would make sense to show that the 0xFFFFFFFFFFFFFFFF is there because null is the default value.

Except in the ZAP UI null is represented as a checkbox for "this is null", not 0xFFFFFFFFFFFFFFFF.

> Or let's make it possible to say default="null"

See https://github.com/project-chip/zap/issues/608